openmls/src/key_packages/mod.rs

```diff
@@ -267,6 +267,7 @@ impl KeyPackage {
             config,
             credential, // FIXME
             LeafNodeSource::KeyPackage(Lifetime::default()),
+            Capabilities::default(),
```

Comment:
So we still don't set them. Let's do a follow up to allow actually passing in values.

Reply:
Yeah. I tried to not touch key packages here. Will do a follow-up :+1:

Reply:
https://github.com/openmls/openmls/issues/1195